### DISCUSSION TITLE
Using Bash Plugin, unable to execute python script using virtualenv

### DISCUSSION THREAD

**Discussion by jbwaclawski** [jhuckaby/Cronicle#476] [Q&A]
Title is the problem.  Here are some code snippets indicating my setup.  The error I keep receiving is that the job is unable to see the directory containing my script files.

Thoughts?

**Bash Script in Cronicle Job**
``` shell
#!/bin/bash

cd /path/to/script/folder/

./scriptname.py
```

**Python Script**
```python
#!.venv/bin/python

print("Hello, world!")
```

**Comment by mikeTWC1984** [comment]
Did you make your py script executable? I just tried on my end and it seemed to work.
Why not just write you script in shell plug right away though?
```python
#!/path/to/your/dir/.venv/bin/python
print("Hello, world!")
```
